sdk/Notifo.SDK/NotifoMobilePush/NotifoMobilePushImplementation.ios.cs

```diff
@@ -48,14 +48,16 @@ public async Task DidReceiveNotificationRequestAsync(UNNotificationRequest reque
         // 3. Enrich with images, which need to be downloaded.
         await EnrichImagesAsync(content, notification);
 
+        await HandlePendingNotificationsAsync();
+
         // 4. Custom enrichment code (could be potentially be expensive).
         EnrichWithCustomCode(content, notification);
     }
 
     /// <inheritdoc />
-    public async Task DidReceivePullRefreshRequestAsync(PullRefreshOptions? options = null)
+    public async Task HandlePendingNotificationsAsync()
     {
-        options ??= new PullRefreshOptions();
+        var options = new PullRefreshOptions();
```

Comment:
This makes the full options useless.

Reply:
yeah that's right. Could be removed completely but what do you think in general of removing the pullRefresh?

Reply:
In general I tried not to have some non-configurable values, but it is weird, that these options are configured over the method. So we should just add a method to INotifoPush like

```
NotifoPush.Current.SetPullOptions(...);
```

Reply:
I have introduced this change in the base branch